setup.py

```diff
@@ -56,7 +56,7 @@ def get_packages(package):
     "httptools>=0.4.0",
     "uvloop>=0.14.0,!=0.15.0,!=0.15.1; " + env_marker_cpython,
     "colorama>=0.4;" + env_marker_win,
-    "watchgod>=0.6",
+    "watchgod==0.10a1",
```

Comment:
The package has been renamed to `watchfiles` as of [v0.10](https://github.com/samuelcolvin/watchfiles/releases/tag/v0.10).

Reply:
I have a horrible memory. Thanks Brendon! :)